SleepEngine/Engine/Clock.h

```diff
@@ -16,6 +16,8 @@ class Clock
     Clock(float fps);
 
     void frameStart();
+	// also sleeps if m_restrictFPS is set and 
+	// the frame was to short to sustain desired fps has passed since last frame
```

Comment:
Please, use 4 whitespaces instead of this W I D E tab.
```suggestion
    // also sleeps if m_restrictFPS is set and
    // the frame was to short to sustain desired fps has passed since last frame
```